service/src/main/java/eu/europa/ec/fisheries/ers/service/mapper/ActivityEntityToModelMapper.java

```diff
@@ -376,33 +377,29 @@ private void mapSpecifiedVesselTransportMeans(FAReportDocument target, Set<Vesse
 
     private void mapSpecifiedContactParties(VesselTransportMeans target, Set<ContactPartyEntity> contactPartyEntities) {
         ArrayList<ContactParty> contactParties = new ArrayList<>();
-        if (CollectionUtils.isNotEmpty(contactPartyEntities)) {
-            for (ContactPartyEntity source : contactPartyEntities){
-                ContactParty contactParty = new ContactParty();
+        for (ContactPartyEntity source : Utils.safeIterate(contactPartyEntities)) {
```

Comment:
Utils is not a good name, maybe SafeIterator.iterate or something like that.

Reply:
I just put the method in an existing Utils class.